src/hooks/pageHooks/useUpdatePageHook.jsx

```diff
@@ -33,13 +37,15 @@ export function useUpdatePageHook(params, queryParams) {
         queryClient.invalidateQueries([PAGE_CONTENT_KEY, { ...params }])
       },
       onSuccess: () => {
-        if (params.collectionName || params.resourceRoomName)
+        if (params.collectionName)
           queryClient.invalidateQueries([
             // invalidates collection pages or resource pages
             DIR_CONTENT_KEY,
             _.omit(params, "fileName"),
           ])
-        else
+        else if (params.resourceRoomName) {
+          queryClient.invalidateQueries(RESOURCE_CATEGORY_CONTENT_KEY)
+        } else
```

Comment:
Same as above, we should include `_.omit(params, "fileName")`!

Reply:
the type here is
```ts 
interface ResourceCategoryRouteParams extends ResourceRoomRouteParams {
  resourceCategoryName: string
}
```
which doesn't have `fileName`!

Reply:
Same as above, we should still be using `queryClient.invalidateQueries([RESOURCE_CATEGORY_CONTENT_KEY, params])` so we don't invalidate other unaffected resource categories!

Reply:
my bad here - missed that and you're totally rgiht